crm_compassion/models/res_partner.py

```diff
@@ -96,6 +96,31 @@ def open_interaction(self):
             "target": "current",
         }
 
+    def open_interaction_full(self):
+        """
+        Populates data for interaction resume and open the view
+        :return: action opening the view
+        """
+        self.ensure_one()
+        self.env["interaction.resume"].populate_resume(self.id, full=True)
+        partners_with_same_email_ids = (
+            self.env["res.partner"]
+                .search([("email", "!=", False), ("email", "=", self.email)])
+                .ids
+        )
+        return {
+            "name": _("Interaction resume"),
+            "type": "ir.actions.act_window",
+            "res_model": "interaction.resume",
+            "view_type": "form",
+            "view_mode": "tree,form",
+            "domain": [("partner_id", "in",
+                        self.ids + partners_with_same_email_ids +
+                        self.other_contact_ids.ids)],
+            "target": "current",
+        }
```

Comment:
This code duplicates the open_interaction method. Why not just calling the other?
```suggestion
        return self.populate_resume(self.id, full=True)
```